src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
```

Comment:
The `<html>` tag should not be self-closing. It should be written as `<html lang="en" class="page">` and closed with a separate `</html>` tag at the end of the document.